cli/cmd/lql_update.go

```diff
@@ -0,0 +1,72 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2020, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/lacework/go-sdk/api"
+)
+
+const (
+	lqlUpdateDebugMsg   string = "updating LQL query"
+	lqlUpdateSuccessMsg string = "LQL query (%v) updated successfully.\n"
```

Comment:
General Go question. Any reason to use `(%v)` here rather than being explicit with the type? eg. `(%s)`

Reply:
`%s` is probably better.  My use of `%v` is likely a result of the tutorials i was watching where they wanted to easily interpolate arbitrary data types...

Reply:
Fixed.